lms/static/scripts/frontend_apps/components/JSTORPicker.js

```diff
@@ -181,7 +187,7 @@ export default function JSTORPicker({ onCancel, onSelectURL }) {
             />
           )}
         </Thumbnail>
-        <div className="space-y-2 grow">
+        <div className="space-y-2 grow flex flex-col">
```

Comment:
The columnar flex styling added in these changes is related to getting the checkbox to be reliably positioned at the bottom of the available main content space, just above the action buttons.